### PR TITLE
Add documentation on FS & BEM surface creation

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

**Comment by hoechenberger** [comment]
Doc artifact: https://1554-167957354-gh.circle-artifacts.com/0/site/getting_started/freesurfer.html

File: docs/source/getting_started/freesurfer.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+BIDS raw datasets, however, do **not** include any of these 3D representations
+and parcellations. (Note that, however, these *derivatives* **are** sometimes
+distributed along with a datasets inside a `derivatives/` folder). Instead,
+they ship e.g. with T1-weighted images only (and, sometimes, include FLASH
+images too).
+
+## Install FreeSurfer
+
+Before running the source-analysis parts of the Study Template, you need to
+create said 3D surfaces and parcellations. This is done using the tool
```
Comment (agramfort) [inline docs/source/getting_started/freesurfer.md:20]: Said?

**Review by agramfort — APPROVED** [review]
Just a misc stuff

File: docs/source/getting_started/freesurfer.md (hunk 1 above)
Comment (hoechenberger) [reply docs/source/getting_started/freesurfer.md:20]: Past of "say" :) as in, "previously mentioned"
Will rephrase as it doesn't seem to be clear!

**Pull request merged**